projects/hslayers/src/components/map/map.service.ts

```diff
@@ -927,22 +1005,35 @@ export class HsMapService {
     }
   }
 
-  simpleImageryProxy(image, src, app: string) {
-    if (src.indexOf(this.HsConfig.get(app).proxyPrefix) == 0) {
+  /**
+   * @public
+   * @param image -
+   * @param src -
+   * @param app - App identifier
+   *
+   */
+  simpleImageryProxy(image, src, app: string): void {
+    if (src.indexOf(this.hsConfig.get(app).proxyPrefix) == 0) {
       image.getImage().src = src;
     } else {
-      const laymanEp = this.HsConfig.get(app).datasources?.find(
-        (ep) => ep.type == 'layman'
-      );
+      const laymanEp = this.hsConfig
+        .get(app)
+        .datasources?.find((ep) => ep.type == 'layman');
       if (laymanEp && src.startsWith(laymanEp.url)) {
         this.laymanWmsLoadingFunction(image, src);
       } else {
-        image.getImage().src = this.HsUtilsService.proxify(src, app); //Previously urlDecodeComponent was called on src, but it breaks in firefox.
+        image.getImage().src = this.hsUtilsService.proxify(src, app); //Previously urlDecodeComponent was called on src, but it breaks in firefox.
       }
     }
   }
 
-  laymanWmsLoadingFunction(image, src: string) {
+  /**
+   * @public
+   * @param image -
+   * @param src -
+   *
+   */
```

Comment:
💔 but these empty comments are just useless lines of code..

Reply:
I know, but I was not sure about what to write there, would appreciate some additional descriptions. :)

Reply:
Hmm.. I will take a look at the map.service and will try to add something.

Reply:
A commit is there 😉